src/tests/getPersonaAttributes/getPersonaAttributes.test.ts

```diff
@@ -1,37 +1,54 @@
+// tslint:disable:no-magic-numbers
+// tslint:disable:max-file-line-count
 import * as assert from 'assert';
 import setup from '../utils/setup';
 import {
   TEST_ORGANISATION,
   TEST_PERSONA_ID,
+  TEST_PERSONA_ID_2,
 } from '../utils/values';
 
 describe('getPersonaAttributes', () => {
   const personaId = TEST_PERSONA_ID;
   const service = setup();
+  const attribute1 = {
+    key: 'theKey',
+    organisation: TEST_ORGANISATION,
+    personaId,
+    value: 'theValue',
+  };
+  const attribute2 = {
+    key: 'theKey2',
+    organisation: TEST_ORGANISATION,
+    personaId,
+    value: true,
+  };
+  const attribute3 = {
+    key: 'theKey3',
+    organisation: TEST_ORGANISATION,
+    personaId: TEST_PERSONA_ID_2,
+    value: true,
+  };
 
-  it('should get all persona attribute', async () => {
-    const attribute1 = {
-      key: 'theKey',
-      organisation: TEST_ORGANISATION,
-      personaId,
-      value: 'theValue',
-    };
-    const attribute2 = {
-      key: 'theKey2',
-      organisation: TEST_ORGANISATION,
-      personaId,
-      value: true,
-    };
-
+  beforeEach(async () => {
+    await service.clearService();
```

Comment:
This is actually done as part of the setup function.

Reply:
As far as I can see, setup is only called once in the describe. Should I be calling setup instead of clearService?

Reply:
I guess it's not actually needed as a beforeEach because the API is readonly and every test uses the same data.